domains/certificates/Query.API/API.IntegrationTests/Testcontainers/RegistryFixture.cs

```diff
@@ -0,0 +1,77 @@
+extern alias registryConnector;
+using System;
+using System.Text;
+using System.Threading.Tasks;
+using DotNet.Testcontainers.Builders;
+using DotNet.Testcontainers.Containers;
+using ProjectOrigin.HierarchicalDeterministicKeys;
+using ProjectOrigin.HierarchicalDeterministicKeys.Interfaces;
+using Xunit;
+
+namespace API.IntegrationTests.Testcontainers;
+
+public class RegistryFixture : IAsyncLifetime
+{
+    private const string registryImage = "ghcr.io/project-origin/registry-server:0.2.0-rc.17";
+    private const string electricityVerifierImage = "ghcr.io/project-origin/electricity-server:0.2.0-rc.17";
+    protected const int GrpcPort = 80;
+    private const string area = "DK1"; //TODO: What about DK2?
+    private const string registryName = "TestRegistry";
+
+    private readonly Lazy<IContainer> registryContainer;
+    private readonly IContainer verifierContainer;
+
+    public string IssuerArea => area;
+    public string RegistryName => registryName;
+    public IPrivateKey IssuerKey { get; init; }
+    public string RegistryUrl => $"http://{registryContainer.Value.Hostname}:{registryContainer.Value.GetMappedPublicPort(GrpcPort)}";
+    protected string RegistryContainerUrl => $"http://{registryContainer.Value.IpAddress}:{GrpcPort}";
+
+    public RegistryFixture()
+    {
+        IssuerKey = Algorithms.Ed25519.GenerateNewPrivateKey();
+
+        verifierContainer = new ContainerBuilder()
```

Comment:
Det kan godt være jeg spørger dumt, men det er bare for forståelse - det her er docker vi starter, right?